winui/Scheduler/Themes.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: Themes in WinUI Scheduler control | Syncfusion
+description: Learn here all about to use various themes in Syncfusion WinUI Scheduler(SfScheduler) control and more.
```

Comment:
Learn here all about to use various themes in Syncfusion WinUI **Scheduler (SfScheduler)** control and more.

Reply:
Highlighted the words.